### ISSUE TITLE
Save Mapbox static api response to png file with node sdk js

### ISSUE THREAD

**Issue by biaozeebub** [mapbox/mapbox-sdk-js#317]
I have the same problem as this Stackoverflow question I found which I'm reproducing for your benefit here:

https://stackoverflow.com/questions/54117885/save-mapbox-static-api-response-to-png-file-with-node-sdk-js

The png file created with the following code using mapbox-sdk-js is unreadable, yet the response headers display content "image/png" and status code is 200. This is not a problem with my token or else, if I use curl with the same parameters I get a proper PNG file (and a nice map).

Could you help me to save the response to a file ? Thanks !

```javascript
const mbxStatic = require('@mapbox/mapbox-sdk/services/static')
const fs = require('fs')

// Get config
const MAPBOX_TOKEN = process.env.MAPBOX_TOKEN

const staticClient = mbxStatic({ accessToken: MAPBOX_TOKEN })

staticClient
  .getStaticImage({
    ownerId: 'mapbox',
    styleId: 'streets-v10',
    width: 200,
    height: 300,
    position: {
      coordinates: [-77, 38],
      zoom: 5
    }
  })
  .send()
  .then(response => {
    const image = response.body
    console.log(response.headers)
    fs.writeFileSync("./map.png", image, 'binary')
  }).catch(error => console.log(error));
```

**Comment by andrewharvey** [comment]
proposed a fix at #351

**Comment by jamesdixon** [comment]
oh man, after 4 hours of trying to get this to work, i just came across this... 😢
